public_app/pages/MoviesDetails.js

```diff
@@ -0,0 +1,114 @@
+import React    from 'react';
+import BigImg   from '../components/BigImg';
+import Nav      from '../components/Nav';
+import Details  from '../components/Details';
+import LogOut   from '../components/LogOut';
+import axios    from 'axios';
+var Link = require ('react-router-dom').Link;
+
+
+
+
+class MoviesDetails extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            item: {},
+            chat: [],
+            chatInput: {
+                author: '',
+                comment: ''
+            }
+        };
+        console.log(this.state);
+        axios.get(`/api/movies/${this.props.match.params.id}`
+                 ).then((response) => {
+            this.setState({
+                item: response.data
+            });
+        });
+        axios.get(`/api/comment/${this.props.match.params.id}`
+                 ).then((response) => {
+            this.setState({
+                chat: response.data
+            });
+        });
+    }
+    handleApi(){
+        axios.get(`/api/comment/${this.props.match.params.id}`
+                 ).then((response) => {
+            this.setState({
+                chat: response.data
+            });
+        });
+    }
+    handleClick(event){
+        let comment = {
+            movieId: this.props.match.params.id,
+            user: this.state.chatInput.author,
+            comment: this.state.chatInput.comment
+        };
+        axios.post(`/api/comment/${this.props.match.params.id}`, comment);
+
+        this.setState({
+            chatInput: {
+                author: '',
+                comment: ''
+            }
+        })
+        this.handleApi()
```

Comment:
You should really think about function naming as these function names does not describe anything, what api where how? What it actually does with the api? http://mojones.net/how-to-pick-bad-function-and-variable-names.html
https://rainsoft.io/coding-like-shakespeare-practical-function-naming-conventions/